app/Http/Transformers/PostTransformer.php

```diff
@@ -27,6 +27,7 @@ public function transform(Post $post)
                 ],
                 'caption' => $post->content->caption,
                 'status' => $post->content->status,
+                'total_reactions' => count($post->content->reactions),
```

Comment:
@DFurnes got it! I'd love to pair if you have time! I'm free so let me know when works best for you :)